python/GafferDispatch/LocalDispatcher.py

```diff
@@ -295,7 +295,10 @@ def __doBackgroundDispatch( self, batch ) :
 			while process.poll() is None :
 
 				if batch.blindData().get( "killed" ) :
-					os.killpg( process.pid, signal.SIGTERM )
+					if os.name == "nt" :
+						subprocess.Popen( "TASKKILL /F /PID {} /T".format( process.pid ), stdout = sys.stderr )
```

Comment:
Can we use `subprocess.check_call()` here, for a bit of extra modernity and error checking?

Reply:
Fixed in 6470f6fb49276c2a951d0e885dcf2c11f259b074